.github/workflows/e2e_libvirt.yaml

```diff
@@ -89,13 +88,6 @@ jobs:
           # For debugging
           cat libvirt.properties
 
-      - name: Install rust toolchain
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: ${{ env.RUST_VERSION }}
-          override: true
-          profile: minimal
-
```

Comment:
Hey Magnus, I've just realised that we still need the rust toolchain to build the kbs-client, so I don't think we can remove it from the workflow yet, unless there is a cached binary there we can use (which was discussed, but I don't think has been done yet)? I'm not sure our `.tools.rust` is required though as it looks like trustee uses a different version, but it's hard-coded in the workflow, so I'm not sure it's easy to get at.

Reply:
oh, you're right. there is a kbs-client being published, though. can we use this?

https://github.com/confidential-containers/trustee/pkgs/container/staged-images%2Fkbs-client

Reply:
looks good:

```
oras pull ghcr.io/confidential-containers/staged-images/kbs-client:sample_only-x86_64-linux-gnu-7df333dba5550571ec2999018ae33e46068ba586
Downloading eebf628a6f2b kbs-client
Downloaded  eebf628a6f2b kbs-client
Pulled [registry] ghcr.io/confidential-containers/staged-images/kbs-client:sample_only-x86_64-linux-gnu-7df333dba5550571ec2999018ae33e46068ba586
Digest: sha256:b4f090dc0af970badc17ed73c6871f3f3b2f2345726745b39b6b0a4d37cb9317
magnuskulke@DESKTOP-2KAPMGM:~/tmp/oras$ ls
kbs-client
magnuskulke@DESKTOP-2KAPMGM:~/tmp/oras$ chmod +x kbs-client
magnuskulke@DESKTOP-2KAPMGM:~/tmp/oras$ ./kbs-client
A command line client tool for KBS APIs.

Usage: kbs-client [OPTIONS] <COMMAND>

Commands:
  config        Set and Config KBS
  get-resource  Get confidential resource
  attest        Attestation and get attestation results token
  help          Print this message or the help of the given subcommand(s)

Options:
      --url <URL>              The KBS server root URL [default: http://127.0.0.1:8080]
      --cert-file <CERT_FILE>  The KBS HTTPS server custom root certificate file path (PEM format)
  -h, --help                   Print help
  -V, --version                Print version
```

Reply:
Nice - it has been done, so I think we switch to that approach then too as part of this. Thanks!

Reply:
I pushed a change to retrieve the kbs-client from oci instead, this means we can also remove the git.kbs entry and should be able to fully de-oxidize the repo.

Reply:
Sorry, I just realised that we also `kustomize` and deploy the KBS from that repo, so I don't think we can get rid of all this code, just the kbs client bit.

Reply:
yup, thx. readded the kbs entry in versions and the cloning step.

looking at versions.yml I'd argue that the split between `KBS_REPO#KBS_VERSION` (kustomize code) and `KBS_IMAGE#KBS_IMAGE_TAG` is probably a bit too contingent. 

there are implicit assumptions that kbs' kustomize scripts w/ version X work with kbs-image-tag version Y and kbs' kustomize "API" doesn't change. there are no guarantees for that. I'd suggest dropping KBS_IMAGE & KBS_IMAGE_TAG including the logic that tweaks those values in the test-provisioner code. CAA should deploy with what is the default in the kbs kustomize manifests (which should be the same revision as KBS_VERSION). In the niche case that someone wants to change that for an experimental build, they would:
- clone the trustee repo
- modify the kustomize manifests
- specify their clone#branch in KBS_REPO#KBS_VERSION

that's not in scope of this PR and probably subjective, but I recommend getting rid of excessive knobs to make the tests more reliable.

Reply:
> looking at versions.yml I'd argue that the split between KBS_REPO#KBS_VERSION (kustomize code) and KBS_IMAGE#KBS_IMAGE_TAG is probably a bit too contingent.

Yeah, I agree and was wondering about this when I realised we deployed from source. It feels like we could just get rid of the test sections like:
```
        log.Infof("Updating kbs image with %q", props["KBS_IMAGE"])
	if err = lio.overlay.SetKustomizeImage("kbs-container-image", "newName", props["KBS_IMAGE"]); err != nil {
		return err
	}

	log.Infof("Updating kbs image tag with %q", props["KBS_IMAGE_TAG"])
	if err = lio.overlay.SetKustomizeImage("kbs-container-image", "newTag", props["KBS_IMAGE_TAG"]); err != nil {
		return err
	}
```
(and all the other related code in the test provisioner) as you suggest and we'd be back to a more stable base.